rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.blocks/10_basic.yml

```diff
@@ -24,7 +24,16 @@
         index: test_index
 
   - do:
-      indices.put_settings:
```

Comment:
Instead of changing the existing test, can you add a new test that uses the remove block API? We don't have any plans yet to remove the existing behavior (of using the settings API), so we still want to cover that in these tests.

Reply:
Also make sure to add a newline at the end of this file when you update it.

Reply:
It sounds better! 👍